Dockerfile

```
@@ -232,8 +232,8 @@ RUN R -e "install.packages('corrplot', dependencies = TRUE)"
 # Install for mutation signature analysis
 RUN R -e "BiocManager::install('ggbio')"
 
-#### Please install your dependencies here
-#### Add a comment to indicate what analysis it is required for
+# package required for shatterseek
+RUN R -e  “remotes::install_github('parklab/ShatterSeek'), dependencies = TRUE”
```

I believe the CI error is because the `)` is in the wrong spot. We'll also want to tie this to a specific commit to the ShatterSeek repository.
```suggestion
RUN R -e  “remotes::install_github('parklab/ShatterSeek', ref = '83ab3effaf9589cc391ecc2ac45a6eaf578b5046', dependencies = TRUE)”
```

---

I updated the Dockerfile as you advised, but it displayed the same error and failed again.  BTW, may I ask what does the `ref = '83ab3effaf9589cc391ecc2ac45a6eaf578b5046'` mean?

---

What source code editor are you using? I've not seen one before that uses the directional quotes (`“`) instead of the standard ones `"`.

---

> What source code editor are you using? I've not seen one before that uses the directional quotes (`“`) instead of the standard ones `"`.

haha, sorry about that, that quote` “` is a quote in the Chinese input method, I didn't notice it.

---

`ref = '83ab3effaf9589cc391ecc2ac45a6eaf578b5046'` is how you tie the installation to a specific commit. `83ab3effaf9589cc391ecc2ac45a6eaf578b5046` is the most recent commit.